Dockerfile.ubuntu

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:14.04
-MAINTAINER Abdeali JK <abdealikothari@gmail.com>
+#MAINTAINER Abdeali JK <abdealikothari@gmail.com>
```

Comment:
Actually I would like to have a 2nd maintainer... does not seam to be possible...